var/spack/repos/builtin/packages/hdf5-vol-async/package.py

```diff
@@ -26,11 +40,11 @@ class Hdf5VolAsync(CMakePackage):
     depends_on("argobots@main")
     depends_on("hdf5@1.13: +mpi +threadsafe")
```

Comment:
we are anticipating 1.13.3 to land soon, but the VOLs will break -- we should look into pinning to 1.13.2 for 11/15 deadline

Reply:
1.13.3 will be released on October 31 (plus or minus a day or two). The VOL struct will break in a few ways:

- The capabilities flag will be expanded to a 64-bit unsigned integer
- You will need to define flags for the VOL tests to run (we have an expanded set now)
- The dataset I/O calls now take arrays in order to support the multi-dataset feature

We'll have a migration guide that tells people how to update their connectors, but targeting 1.13.2 for your 11/15 deadline is probably the right decision.

Reply:
> targeting 1.13.2 for your 11/15 deadline is probably the right decision.

OK thanks for weighing in, right now the VOLs are constrained to need `1.13:1.13.2` and a `NOTE` in `hdf5` to not delete `1.13.2` until we're ready to update all three (the SDK cannot update say `hdf5-vol-async` which would need `>=1.13.3` until the other two are ready, keeping `1.13.2` available).

In the future the versions associated with `1.13` will no longer be possible to build, my understanding is this was the desired goal for 1.14.

Reply:
Yes. 1.13.x releases are essentially release candidates for 1.14.0. We just run them through the main release process like a regular release to make it easier for people to keep up with the next major release of HDF5 and give us early feedback.

Reply:
You'll still be able to check out those branches and build them, but we will probably pull the binaries off of our website when 1.14.0 releases.

Reply:
Minor difference - you'll be able to check out the tags, not the branches;  tags differ from branch names only by hdf5- instead of hdf5_.  e.g.  tag hdf5-1_13_3 instead  of branch hdf5_1_13_3.

Reply:
As long as `version("1.13.2", ...)` remains in `spack` until `ecp-data-vis-sdk` is ready to upgrade (see notes in `hdf5/package.py`) this should be OK.  We will likely be able to do that fairly quickly provided the VOLs here also update and release new versions.